README.md

```diff
@@ -2,66 +2,47 @@
 
 # Hello Microverse
 
-> Description the project.
+> This is the first project given by microverse. This is now uploaded in the GITHUB for review, this project is to understand the process of github.com and develop a project professionally.
 
+:heavy_check_mark: A .html File added
+:heavy_check_mark: A .css file added
+:heavy_check_mark: Linters added
+:heavy_check_mark: .gitignore file added
+:heavy_check_mark: Description for the project is added
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- Major languages : HTML, JavaScript, CSS
+- Frameworks: No frameworks used
+- Technologies used: HTML, CSS, GIT, Linters
```

Comment:
- [OPTIONAL] Good job implementing the changes in your readme file, it looks very professional. There is just one thing I would like to mention. You are not using Javascript in this project, so kindly remove it from your major languages. 😸